tests/androidresource/AndroidResourceTest.java

```diff
@@ -0,0 +1,12 @@
+import org.checkerframework.checker.androidresource.qual.res.*;
+import src.R;
+
+class AndroidResourceTest {
+    AndroidResourceTest() {
+    }
+
+    void Test() {
+        @XmlRes int xml_res = R.xml.file_paths;
+//        xml_res = R.string.file_paths;    // Flagged by CF

```

Comment:
Instead of a comment you must actually make this a test case.
See https://github.com/typetools/checker-framework/blob/master/checker/tests/README for detailed instructions.

Reply:
This will be resolved in #4